src/main/java/org/mdbenefits/app/data/enums/HelpNeededType.java

```diff
@@ -3,24 +3,55 @@
 import static org.mdbenefits.app.data.enums.ProgramType.SNAP;
 import static org.mdbenefits.app.data.enums.ProgramType.TCA;
 import static org.mdbenefits.app.data.enums.ProgramType.TDAP;
-import static org.mdbenefits.app.data.enums.ProgramType.OTHER;
+import static org.mdbenefits.app.data.enums.ProgramType.RCA;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
 
 import lombok.Getter;
 
 @Getter
 public enum HelpNeededType {
 
-  FOOD( "help-needed.food", SNAP),
-  CHILDREN("help-needed.children", TCA),
-  //UTILITIES("help-needed.utilities", OHEP),
-  DISABILITY("help-needed.disability", TDAP),
-  NOT_SURE("help-needed.not-sure", OTHER);
+    FOOD("help-needed.food", SNAP, new ArrayList<>(List.of(SNAP, TCA))),
```

Comment:
Kept Program Type as the key that determines what we would want to have pre-checked. relevant programs determines the potential options that will appear to the final user.